sections/dashboard/Transactions/TradeHistory/TradeHistory.tsx

```diff
@@ -149,6 +153,7 @@ const TradeHistory: FC<TradeHistoryProps> = ({ trades, isLoading, isLoaded }) =>
 			]}
 			columnsDeps={columnsDeps}
 			data={trades}
+			hiddenColumns={[isL2 ? 'timestamp' : '']}
```

Comment:
Not very likely, but this might cause some issues if there's a column with an empty key. This might be a bit better:

```suggestion
			hiddenColumns={isL2 ? ['timestamp'] : undefined}
```

Reply:
Thanks for your comments. I tested the change locally and got the following error. Could you please check it once more? 
![error](https://user-images.githubusercontent.com/4819006/153286043-31638061-008f-4745-bc66-3e2dc6dd8f81.JPG)
r